azuredevops/internal/acceptancetests/resource_serviceendpoint_aws_test.go

```diff
@@ -0,0 +1,49 @@
+// +build all resource_serviceendpoint_aws
+// +build !exclude_serviceendpoints
```

Comment:
@phillebaba  Can you help add more acceptance tests to cover more scenario.  You can refer this resource [bitbucket](https://github.com/terraform-providers/terraform-provider-azuredevops/blob/master/azuredevops/internal/acceptancetests/resource_serviceendpoint_bitbucket_test.go)

Reply:
I added the same type of tests that were in the bitbucket acc test.